package.json

```diff
@@ -23,6 +26,7 @@
         "express": "^4.17.1",
         "express-fileupload": "^1.2.1",
         "express-jwt": "^7.7.5",
+        "fluent-audiowaveform": "^1.0.8",
```

Comment:
Is this used ? 

Also Add in the README information about how to build the audiowave for local usage